SF_Users_API/db/seeds/dev/users.js

```diff
@@ -0,0 +1,35 @@
+
+exports.seed = function(knex, Promise) {
```

Comment:
Do we need to seed this database?  I think it might be unnessessary since the handoff from OAuth won't recognize this data and find the right user in the database unless we add in the right OAuth id we getback.